Fabric.Identity.API/scripts/Install-IdPSS-Utilities.psm1

```diff
@@ -92,13 +92,18 @@ function New-FabricAzureADApplication() {
         $app = New-AzureADApplication -Oauth2AllowImplicitFlow $true -RequiredResourceAccess $permission -DisplayName $appName -ReplyUrls $replyUrls.name -AvailableToOtherTenants $isMultiTenant -GroupMembershipClaims $groupMembershipClaims
     }
     else {
-        Set-AzureADApplication -ObjectId $app.ObjectId -RequiredResourceAccess $permission -Oauth2AllowImplicitFlow $true -ReplyUrls $replyUrls.name -AvailableToOtherTenants $isMultiTenant -GroupMembershipClaims $groupMembershipClaims
+        # Do not overwrite, append to existing urls
+        $existingUrls = $app.ReplyUrls
+        foreach($replyUrl in $replyUrls.name) {
+            $existingUrls.Add($replyUrl)
```

Comment:
If a duplicate replyUrl is added, when the call is made to update the application, then the update fails.  Need to find a way to filter out existing replyUrls